### PR TITLE
fix(INJI-397)error message for deleted vcfile

### PR DESCRIPTION
**Pull request by srikanth716**
[tamperedVcs.webm](https://github.com/mosip/inji/assets/97477121/a92e71dd-25f4-491b-94b8-11a37a1d8fe3)

### CODE REVIEW

File: machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts
Comment (tilak-puli) [inline machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts:74]: two doubts 

1. won't vcKey will be same as vcKey in metadata. if it is, why to get it from store and store it separately in context instead of using vcMetadata
2. if it is not same vcKey, please name it appropriate to signify that this VcKey of tempered vc

File: machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts
Comment (tilak-puli) [inline machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts:1207]: context already has vcMetadata, we can use it instead of creating again here

File: machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts
Comment (tilak-puli) [inline machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts:1208]: can we create a enum for Activity log types instead of hardcoding here?

File: components/ActivityLogText.tsx
Comment (tilak-puli) [inline components/ActivityLogText.tsx:19]: doubt: do we have any activities without label?

File: machines/store.ts
Comment (tilak-puli) [inline machines/store.ts:398]: is this condition duplicate of tampered condition. can we club these two cases?

File: shared/storage.ts
Comment (tilak-puli) [inline shared/storage.ts:95]: duplicate of 98

File: shared/storage.ts
Comment (tilak-puli) [inline shared/storage.ts:108]: if both the file and hmac is deleted by attacker, won't this put our app in a state where vc card is stuck at loading state

File: components/VC/ExistingMosipVCItem/ExistingMosipVCItem.tsx
Comment (tilak-puli) [inline components/VC/ExistingMosipVCItem/ExistingMosipVCItem.tsx:126]: if we have this overlay in vc item components, will there be a chance that multiple overlays are shown at a time if multiple VC's are tampered at a time

**Review by tilak-puli — CHANGES_REQUESTED** [review]

File: machines/vc.ts
Comment (tilak-puli) [inline machines/vc.ts:328]: as myVcs is a list of objects. calling includes function might not work well with reference. Please use isEquals function.

and also using vc as name of parameter is confusing as its vcMetadata

**Review by swatigoel — APPROVED** [review]
Approving so that merge button becomes enabled (reviewed by @tilak-puli )

**Pull request merged**
